docs/reference/query-dsl/script-score-query.asciidoc

```diff
@@ -4,18 +4,19 @@
 <titleabbrev>Script score</titleabbrev>
 ++++
 
-The `script_score` allows you to modify the score of documents that are
-retrieved by a query. This can be useful if, for example, a score
-function is computationally expensive and it is sufficient to compute
-the score on a filtered set of documents.
+Uses a <<modules-scripting,script>> to modify the <<relevance-scores,relevance
+scores>> of another query.
```

Comment:
This is only one use case, but other use cases don't need to use relevance scores from another query. I would rephrase it something like this:

Use <<modules-scripting,script>> to provide/supply a custom score for documents.

Reply:
Fixed with a31f4a8ae60f8bbae76f211f53e1e88ee1a49541.